Update README and metadata (#1639)
voxpupuli/puppet-nginx: README.md

This module was migrated from James Fryman <james@frymanet.com> to Vox Pupuli.

> ## WARNING: This module is no longer supported for Puppet Enterprise or Puppet Core

Reviewer: ```suggestion
## WARNING: This module is no longer tested on Puppet Enterprise or Puppet Core
```

I think that I've come around to @tuxmea's suggestion. "Supported" is such an overloaded word, in some cases we use it to mean "this works on," in some cases we use it to mean "we've tested it on," in some cases we use it to mean "there's commercial support for," and so on.

Let's just be specific and say the accurate thing we're getting at.

Author: Oops, I intended to but neglected to change the header :)

Author: Fixed
